src/librustc_mir/dataflow/drop_flag_effects.rs

```diff
@@ -231,8 +231,13 @@ pub(crate) fn drop_flag_effects_for_location<'a, 'gcx, 'tcx, F>(
                     }
                 }
             }
+            mir::StatementKind::StorageDead(local) => {
```

Comment:
I don't think this is needed - we already iterate over move-outs the statement before.